MIT-LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Sorry if I wasn't clear originally, but GitHub [expects](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) the file to be named simply LICENSE, or LICENSE.md.